<launch>
  
  <node pkg="lidar_process" name="lidar_process_node" type="lidar_process_node" output="screen"  ns="FrontLidar">
    <param name="crop_x_min" value="0.2"/>
    <param name="crop_x_max" value="10.0"/>
    <param name="crop_y_min" value="-2.5"/>
    <param name="crop_y_max" value="2.5"/>
    <param name="crop_z_min" value="-1.5"/>
    <param name="crop_z_max" value="3.5"/>
    <param name="cluster_tolerance" value="0.3"/>
    <param name="cluster_min_size" value="10"/>
    <param name="cluster_max_size" value="70000"/>
    <param name="frame_id" value="front_laser_link"/>
    <param name="pointcloud_sub_topic" value="/FrontLidar/lslidar_point_cloud"/>
    <param name="obstacle_pub_topic" value="/front/obstacles"/>
  </node>

  <node pkg="lidar_process" name="lidar_process_node" type="lidar_process_node" ns="BackLidar">
    <param name="crop_x_min" value="0.2"/>
    <param name="crop_x_max" value="10.0"/>
    <param name="crop_y_min" value="-2.5"/>
    <param name="crop_y_max" value="2.5"/>
    <param name="crop_z_min" value="-1.5"/>
    <param name="crop_z_max" value="3.5"/>
    <param name="cluster_tolerance" value="0.3"/>
    <param name="cluster_min_size" value="10"/>
    <param name="cluster_max_size" value="70000"/>
    <param name="frame_id" value="back_laser_link"/>
    <param name="pointcloud_sub_topic" value="/BackLidar/lslidar_point_cloud"/>
    <param name="obstacle_pub_topic" value="/back/obstacles"/>
  </node>

<!--
  <node pkg="tf" type="static_transform_publisher" name="world_lidar" args="6.2 0 0 0 0 0 /world /front_laser_link 100" />
-->
</launch>
